Actions/RunPipeline/RunPipeline.ps1

```diff
@@ -236,6 +237,10 @@ try {
     # Analyze app.json version dependencies before launching pipeline
 
     # Analyze InstallApps and InstallTestApps before launching pipeline
+    if ($settings.runTestsInAllInstalledTestApps) {
+        $install.TestApps = ConvertTo-UnwrappedTestApps -TestApps $install.TestApps
```

Comment:
Why not keep it simple: 

E.g.
```
$install.TestApps = $install.TestApps | % { $_.TrimStart("(").TrimEnd(")") }
```

With an overexplaining comment why this way the tests in these apps will run.